examples/package.json

```diff
@@ -0,0 +1,19 @@
+{
+    "private": true,
+    "workspaces": [
+        "stable/test/end_to_end/candid_rpc/async_await",
```

Comment:
Shouldn't we be testing more stable packages than experimental packages?

Reply:
we don't actually want to add any more